yodeploy/cmds/install_certificates.py

```diff
@@ -0,0 +1,177 @@
+#!/usr/bin/env python
+from __future__ import print_function
+
+import argparse
+from cgi import parse_header
+import os
+import stat
+import sys
+
+from demands import HTTPServiceClient
+
+sys.path.insert(0, os.path.join(os.path.dirname(__file__), '..', '..'))
+
+from yodeploy import config  # noqa
+
+# overrwrite the unsafe python 2 input method with raw_input, which does not
+# eval user input. In Python 3, raw_input has been renamed to input. This
+# assignment will raise a NameError, doing nothing.
+try:
```

Comment:
We're doing this in at least one other place.

six is currently a transitive dependency. I don't mind making that explicit and using `six.moves.input` here